sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -167,10 +160,11 @@ export interface CertificatePollerOptions extends coreHttp.OperationOptions {
 
 // @public
 export interface CertificateProperties {
+    certificatePolicy?: CertificatePolicy;
     readonly createdOn?: Date;
     enabled?: boolean;
     readonly expiresOn?: Date;
-    id?: string;
+    readonly id?: string;
     name?: string;
```

Comment:
Made this readonly since this property is only assigned by the server. .Net never sends this, but they do also use the class to extend the optional parameters of the relevant methods.